README.md

```diff
@@ -1,6 +1,50 @@
-# curriculum-cr-training-mobile-menu
+# Portfolio Project
 
+> This project contains the portfolio worked on during the Microverse Code Reviewer Training.
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> I worked on the Mobile Menu.
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+# Screenshot
+## Mobile View
+
+Width - 375px
+
+![Mobile view](./screenshot.png)
+
+
+![Open Mobile Menu](./screenshot2.png)
```

Comment:
[Optional] Could you consider moving your screenshot files(screenshot.png and screenshot1.png) into the media folder, as this will give your project folder a professional look? You can also apply this to every other media file in your project.
![image](https://user-images.githubusercontent.com/62849872/142717559-59f4e415-f3fa-4cd2-8a55-ee486fe429d5.png)